### PR TITLE
Replace restricted strings in app name and add random-route

### LINKED ISSUE #394: Application name must not contain "_"

### PR DESCRIPTION
**Pull request by JensMueller2709**
resolves #394

### CODE REVIEW

File: server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java [hunk 1]
```diff
@@ -222,6 +233,6 @@ private void addProviderServiceOfferings(BashScript deployScript) throws IOExcep
             }
             deployScript.append(String.format("# %-20s %-40s %-50s ", service.getLabel(), plans, service.getDescription()));
         }
-        deployScript.append("\n* These service plans have an associated cost. Creating a service instance will incur this cost.");
+        deployScript.append("\n# * These service plans have an associated cost. Creating a service instance will incur this cost.");
```
Comment (mmunozba) [inline server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java]: Shouldn't this just be `# These service...` instead of `\n# * These service...`

File: server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/Application.java [hunk 2]
```diff
@@ -14,7 +14,10 @@
  */
 public class Application {
 
+    //private static int counter;
```
Comment (mmunozba) [inline server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/Application.java]: Why is this here? Same with the other counter-related additions

Comment (JensMueller2709) [reply server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/Application.java]: already deleted

File: server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java (hunk 1 above)
Comment (JensMueller2709) [reply server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java]: No, I need an additional new line and the `*` is also needed as annotation.
